packages/manager/modules/emailpro/src/domain/emailpro-domain.service.js

```diff
@@ -207,6 +207,12 @@ export default /* @ngInject */ function EmailProDomains(
       .then(({ data }) => data);
   };
 
+  this.getDomain = function getDomain(organization, serviceName, domain) {
+    return this.services.$http
```

Comment:
![image](https://github.com/ovh/manager/assets/8767496/0daf504f-883d-472d-a72e-c1d6edb075c9)

Are you sure that the API is right here?